datasets/race/dataset_infos.json

```diff
@@ -1 +1 @@
\ No newline at end of file
+{"default": {"description": "Race is a large-scale reading comprehension dataset with more than 28,000 passages and nearly 100,000 questions. The\n dataset is collected from English examinations in China, which are designed for middle school and high school students.\nThe dataset can be served as the training and test sets for machine comprehension.\n\n", "citation": "@article{lai2017large,\n    title={RACE: Large-scale ReAding Comprehension Dataset From Examinations},\n    author={Lai, Guokun and Xie, Qizhe and Liu, Hanxiao and Yang, Yiming and Hovy, Eduard},\n    journal={arXiv preprint arXiv:1704.04683},\n    year={2017}\n}\n", "homepage": "http://www.cs.cmu.edu/~glai1/data/race/", "license": "", "features": {"article": {"dtype": "string", "id": null, "_type": "Value"}, "answer": {"dtype": "string", "id": null, "_type": "Value"}, "question": {"dtype": "string", "id": null, "_type": "Value"}, "options": {"feature": {"dtype": "string", "id": null, "_type": "Value"}, "length": -1, "id": null, "_type": "Sequence"}}, "supervised_keys": null, "builder_name": "race", "config_name": "default", "version": {"version_str": "0.1.0", "description": null, "datasets_version_to_prepare": null, "major": 0, "minor": 1, "patch": 0}, "splits": {"test": {"name": "test", "num_bytes": 8775394, "num_examples": 4934, "dataset_name": "race"}, "train": {"name": "train", "num_bytes": 157308694, "num_examples": 87866, "dataset_name": "race"}, "validation": {"name": "validation", "num_bytes": 8647200, "num_examples": 4887, "dataset_name": "race"}}, "download_checksums": {"http://www.cs.cmu.edu/~glai1/data/race/RACE.tar.gz": {"num_bytes": 25443609, "checksum": "b2769cc9fdc5c546a693300eb9a966cec6870bd349fbc44ed5225f8ad33006e5"}}, "download_size": 25443609, "dataset_size": 139047912, "size_in_bytes": 227675112}}
```

Comment:
Could you update the dataset_info please ? So that it includes the new configurations

To do so, please delete this file and regenerate it with 
```
datasets-cli test ./race --save_infos --all_configs
```

Reply:
sure! 👍

Reply:
Done in here d6fbac0 @lhoestq 

However, I was wondering the following: I just tested it locally and noticed that if I tried to use the lib as:
`datasets.load_dataset('race')` instead of `datasets.load_dataset('race', 'all')` it still throws the checksum error :( 

I was expecting some kind of warning/error logging that I should pass either `all`; `high` or `middle`. Have I done something wrong?

Could you please advise me? Then I think that I could fix it :)

Reply:
`datasets.load_dataset('race')` loads the dataset script that is on master if you installed the project using git clone.
It you try `datasets.load_dataset("./datasets/race")` however it should run the local script that you changed

Reply:
cool! Thanks 👍 

Just check it again. Everything seems ok by now.